src/assets/i18n/en.json5

```diff
@@ -5454,4 +5444,5 @@
 
   "vocabulary-treeview.search.form.add": "Add",
 
+  "search.filters.filter.entityType.text": "Item Type",
```

Comment:
Please move this filter setting up alongside all the others.  We want to keep them all grouped together to make them easy to find.